lib/resources.js

```diff
@@ -270,6 +271,7 @@ module.exports = function (app, _private = null) {
           .then((resp) => {
             if (checkForNestedHitsAndSource(resp)) {
               resp.hits.hits[0]._source.items = resp.hits.hits[0].inner_hits.items.hits.hits.map((item) => item._source)
+              resp.hits.hits[0]._source.electronicResources = resp.hits.hits[0].inner_hits.electronicResources.hits.hits.map((resource) => resource._source)
```

Comment:
the util function checkForNestedHitsAndSource specifically checks for inner_hits.items.hits.hits . perhaps we want to give it an argument so that it can check for inner_hits.electronicResources.hits.hits. otherwise, this might want some separate input validation.

Reply:
👍